libbeat/processors/add_kubernetes_metadata/config.go

```diff
@@ -43,11 +48,33 @@ type Enabled struct {
 
 type PluginConfig []map[string]common.Config
 
+func getSystemKubeConfig() string {
+	envKubeConfig := os.Getenv("KUBECONFIG")
```

Comment:
If this environment variable is used at the end, please use the constant `clientcmd.RecommendedConfigPathEnvVar`.
```suggestion
	envKubeConfig := os.Getenv(clientcmd.RecommendedConfigPathEnvVar)
```

Reply:
Will update we decide to keep envKubeConfig check :)